source/_addonStore/dataManager.py

```diff
@@ -139,15 +154,22 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 		if not os.path.exists(cacheFilePath):
 			return None
 		with open(cacheFilePath, 'r') as cacheFile:
-			cacheData = json.load(cacheFile)
-		if not cacheData:
+			try:
+				cacheData = json.load(cacheFile)
+			except Exception:
+				return None

```

Comment:
I think this should log at exception level

```suggestion
				log.exception(f"Invalid add-on store cache")
				return None
```